Robot2019/src/main/java/frc/robot/RobotMap.java

```diff
@@ -18,28 +24,57 @@
  */
 public class RobotMap {
   static WPI_TalonSRX leftMaster, rightMaster;
+
   static WPI_VictorSPX leftSlave1, leftSlave2, rightSlave1, rightSlave2;
+
+  static Encoder leftEnc, rightEnc;
+
   static String driveMode;
 
+  static ArrayList<WPI_TalonSRX> masters;
+  static ArrayList<WPI_VictorSPX> slaves;
+  static ArrayList<Encoder> encoders;
```

Comment:
Same as joysticks; is there a reason to have these in an ArrayList? If not, then it would be better to have each one separate.